ga/latest/full/helpers/build/features-installed.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Is the only difference between this file and the one in `kernel` the warning? wondering if it would be okay to output the warning in both cases?

Reply:
Yes, the only difference is the warning message was suppressed.